src/test/java/seedu/address/logic/commands/EditSellerDescriptorTest.java

```diff
@@ -37,35 +37,35 @@ public void equals() {
         assertFalse(DESC_AMY.equals(DESC_BOB));
 
         // different name -> returns false
-        EditPersonDescriptor editedAmy = new EditPersonDescriptorBuilder(DESC_AMY).withName(VALID_NAME_BOB).build();
+        EditSellerDescriptor editedAmy = new EditSellerDescriptorBuilder(DESC_AMY).withName(VALID_NAME_BOB).build();
         assertFalse(DESC_AMY.equals(editedAmy));
 
         // different phone -> returns false
-        editedAmy = new EditPersonDescriptorBuilder(DESC_AMY).withPhone(VALID_PHONE_BOB).build();
+        editedAmy = new EditSellerDescriptorBuilder(DESC_AMY).withPhone(VALID_PHONE_BOB).build();
         assertFalse(DESC_AMY.equals(editedAmy));
 
         // different email -> returns false
-        editedAmy = new EditPersonDescriptorBuilder(DESC_AMY).withEmail(VALID_EMAIL_BOB).build();
+        editedAmy = new EditSellerDescriptorBuilder(DESC_AMY).withEmail(VALID_EMAIL_BOB).build();
         assertFalse(DESC_AMY.equals(editedAmy));
 
         // different address -> returns false
-        editedAmy = new EditPersonDescriptorBuilder(DESC_AMY).withHousingType(VALID_HOUSING_TYPE_BOB).build();
+        editedAmy = new EditSellerDescriptorBuilder(DESC_AMY).withHousingType(VALID_HOUSING_TYPE_BOB).build();
         assertFalse(DESC_AMY.equals(editedAmy));
 
         // different tags -> returns false
-        editedAmy = new EditPersonDescriptorBuilder(DESC_AMY).withTags(VALID_TAG_HUSBAND).build();
+        editedAmy = new EditSellerDescriptorBuilder(DESC_AMY).withTags(VALID_TAG_HUSBAND).build();
         assertFalse(DESC_AMY.equals(editedAmy));
     }
 
     @Test
     public void toStringMethod() {
-        EditPersonDescriptor editPersonDescriptor = new EditPersonDescriptor();
-        String expected = EditPersonDescriptor.class.getCanonicalName() + "{name="
-                + editPersonDescriptor.getName().orElse(null) + ", phone="
-                + editPersonDescriptor.getPhone().orElse(null) + ", email="
-                + editPersonDescriptor.getEmail().orElse(null) + ", housingType="
-                + editPersonDescriptor.getHousingType().orElse(null) + ", tags="
-                + editPersonDescriptor.getTags().orElse(null) + "}";
-        assertEquals(expected, editPersonDescriptor.toString());
+        EditSellerDescriptor editSellerDescriptor = new EditSellerDescriptor();
+        String expected = EditSellerDescriptor.class.getCanonicalName()
+                + "{name=" + editSellerDescriptor.getName().orElse(null)
+                + ", phone=" + editSellerDescriptor.getPhone().orElse(null)
+                + ", email=" + editSellerDescriptor.getEmail().orElse(null)
+                + ", housingType=" + editSellerDescriptor.getHousingType().orElse(null)
```

Comment:
Wait, is the housingType still available under seller? I thought it has been removed already.

Reply:
Eh? No, I don't think your PR touched this file earlier.

I'll update this later together with my seller/buyer tag since I'm removing another field (tag) from this class later.